addons/api/mirage/factories/target.js

```diff
@@ -46,17 +45,7 @@ export default factory.extend({
    */
   withAssociations: trait({
     afterCreate(target, server) {
-      let randomlySelectedHostSets,
-        randomlySelectedCredentialLibraries,
-        randomlySelectedCredentials;
-      randomlySelectedHostSets = server.schema.hostSets
-        // BLERG:  fun fact, for no reason at all, the element passed
-        // into a where function is not a full model instance, as you might
-        // expect at this level of abstraction, but appears to be a serialized
-        // representation of the model instance.  It's very confusing since
-        // the result set of `where` _is a collection of model instances_.
-        .where((hostSet) => hostSet.scopeId === target.scope.id)
-        .models.filter(() => randomBoolean(hostSetChance));
+      let randomlySelectedCredentialLibraries, randomlySelectedCredentials;
```

Comment:
Any reason we removed the random hosts? Don't we still want a random number of hosts for good testing coverage and mirage usage?

I would've expected if we wanted addresses we would have a different trait for that

Reply:
Good point. I wasn't thinking about mirage when I made the change. I will revert this and add a `withAddress` trait.

Reply:
Fixed.